app/index.js

```diff
@@ -107,6 +111,19 @@ var NodeModuleGenerator = yeoman.Base.extend({
                 type: 'input',
                 name: 'username',
                 message: 'What is your GitHub username?',
+                store: true,
+                validate: function(input) {
+                    var res =   !input.startsWith('_')   &&
```

Comment:
It would've been premature optimization if it would ever need to change. GitHub's rules are not going to change.